src/branding/migrations/0004_add_swedish_footer.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.1.6 on 2021-02-16 08:38
```

Comment:
It would be good to add another migration that copies the content from the english footer to the swedish footer so that websites don't end up with empty footers on the swedish versions